tsfile/src/main/java/org/apache/iotdb/tsfile/common/serialization/ChunkHeaderSerializer.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iotdb.tsfile.common.serialization;
+import org.apache.iotdb.tsfile.file.MetaMarker;
+import org.apache.iotdb.tsfile.file.header.ChunkHeader;
+import org.apache.iotdb.tsfile.file.metadata.enums.CompressionType;
+import org.apache.iotdb.tsfile.file.metadata.enums.TSDataType;
+import org.apache.iotdb.tsfile.file.metadata.enums.TSEncoding;
+import org.apache.iotdb.tsfile.utils.ReadWriteIOUtils;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.ByteBuffer;
+
+public class ChunkHeaderSerializer implements IDataSerializer<ChunkHeader, Boolean> {
+
+    @Override
+    public int serializeTo(ChunkHeader data, OutputStream outputStream) throws IOException {
+        int length = 0;
+        length += ReadWriteIOUtils.write(MetaMarker.CHUNK_HEADER, outputStream);
+        length += ReadWriteIOUtils.write(data.getMeasurementID(), outputStream);
+        length += ReadWriteIOUtils.write(data.getDataSize(), outputStream);
+        length += ReadWriteIOUtils.write(data.getDataType(), outputStream);
+        length += ReadWriteIOUtils.write(data.getNumOfPages(), outputStream);
+        length += ReadWriteIOUtils.write(data.getCompressionType(), outputStream);
+        length += ReadWriteIOUtils.write(data.getEncodingType(), outputStream);
+        return length;
+    }
+
+    @Override
+    public int serializeTo(ChunkHeader data, ByteBuffer buffer) {
+        int length = 0;
+        length += ReadWriteIOUtils.write(MetaMarker.CHUNK_HEADER, buffer);
+        length += ReadWriteIOUtils.write(data.getMeasurementID(), buffer);
+        length += ReadWriteIOUtils.write(data.getDataSize(), buffer);
+        length += ReadWriteIOUtils.write(data.getDataType(), buffer);
+        length += ReadWriteIOUtils.write(data.getNumOfPages(), buffer);
+        length += ReadWriteIOUtils.write(data.getCompressionType(), buffer);
+        length += ReadWriteIOUtils.write(data.getEncodingType(), buffer);
+        return length;
+    }
+
+    @Override
+    public ChunkHeader deserializeFrom(ByteBuffer buffer, Boolean options) {
```

Comment:
Better to add javadoc here to explain what the `options` (option) is.

Reply:
Besides, to keep consistent with `deserializeFrom(InputStream ...`, better to move the logic for handling the `offset` of the ByteBuffer in this implementation, rather than the `ChunkerHeader` class

Reply:
How about renaming the parameter option to markerRead